manifests/g/Google/GCPW/70.103.49152/Google.GCPW.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Google.GCPW
+PackageVersion: 70.103.49152
+PackageLocale: en-US
+Publisher: Google LLC
+PackageName: Google Credential Provider for Windows
+License: Proprietary
+ShortDescription: Google Credential Provider for Windows® (GCPW) lets users sign in to Windows® devices with the Google Account they use for work.

```

```suggestion
ShortDescription: Google Credential Provider for Windows® (GCPW) lets users sign in to Windows® devices with the Google Account they use for work.
Moniker: gcpw
```
Adding the moniker will allow users to find this just by using `winget install GCPW`